main/src/main/res/values/strings.xml

```diff
@@ -335,6 +335,7 @@
     <string name="app_no_longer_exists">Package %s is no longer installed, removing it from app allow/disallow list</string>
     <string name="vpn_disallow_radio">VPN is used for all apps but exclude selected</string>
     <string name="vpn_allow_radio">VPN is used for only for selected apps</string>
+    <string name="vpn_allow_bypass">Allow apps to bypass the VPN</string>
```

Comment:
This needs translation, but I only speak English. I don't want to rely on Google Translate for translations.